helm/aws-operator/templates/configmap.yaml

```diff
@@ -45,15 +45,25 @@ data:
           clusterDomain: '{{ .Values.Installation.V1.Guest.Kubernetes.ClusterDomain }}'
           networkSetup:
             docker:
+              {{- if hasPrefix .Values.Installation.V1.Provider.AWS.Region 'cn-' }}
```

Comment:
`hasPrefix` documentation: https://helm.sh/docs/chart_template_guide/function_list/#hasprefix-and-hassuffix